docs/spec/bank/04_events.md

```diff
@@ -0,0 +1,23 @@
+# Events
+
+The bank module emits the following events:
+
+## Handlers
+
+### MsgSend
+
+| Type     | Attribute Key | Attribute Value    |
+|----------|---------------|--------------------|
+| transfer | recipient     | {recipientAddress} |
+| message  | module        | bank               |
+| message  | action        | send               |
+| message  | sender        | {senderAddress}    |
+
+### MsgMultiSend
```

Comment:
do we want to get rid of this message type?

Reply:
Why's that @jackzampolin? We support multi I/O sends.